corehq/apps/export/models/new.py

```diff
@@ -1011,11 +1028,42 @@ def _question_path_to_doc_path(string_path):
     return path
 
 
+def _question_path_to_path_nodes(string_path, repeats):
+    """
+    Return a list of PathNodes suitable for a TableConfiguration or ExportGroupSchema
+    path, from the given path and list of repeats.
+    :param string_path: A path to a question or group, like "/data/group1"
+    :param repeats: A list of repeat groups, like ["/data/repeat1"]
+    :return: A list of PathNodes
+    """
+    if not string_path:
+        return []
+
+    parts = string_path.split("/")
+    assert parts[0] == ""
+    parts = parts[1:]
+
+    repeat_test_string = ""
+    path = []
+    for part in parts:
+        repeat_test_string += "/" + part
+        path.append(PathNode(name=part, is_repeat=repeat_test_string in repeats))
+
+    assert path[0] == PathNode(name="data")
+    path[0].name = "form"
```

Comment:
I believe so, because this function is only used on form questions. The function this replaces does the same thing: https://github.com/dimagi/commcare-hq/commit/a3442403042b7060c3552728d505b83d2bf5dea9#diff-9087195bf99fb8ba2fda775b0556aceaL1010